daphne_worker_test/wrangler.toml

```
@@ -28,6 +28,11 @@ bindings = [
   { name = "DAP_GARBAGE_COLLECTOR", class_name = "GarbageCollector" },
 ]
 
+[[kv_namespaces]]
+  binding = "DAP_HPKE_CONFIG_STORE"
```

Be sure to update `daphne_worker` accordingly.
```suggestion
  binding = "DAP_HPKE_RECEIVER_CONFIG_STORE"
```

---

You're right. I've changed this and the KV name `hpke_config` to `HPKE_RECEIVER_CONFIG` and `hpke_receiver_config`